### PR TITLE
Issue 16 ResNet18

### PR DESCRIPTION
**Pull request by abdksyed**
Added ResNet18 Model.

### CODE REVIEW

File: src/dlutils/models/ResNets/resnet.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+        x = x.view(x.size(0), -1)
+        x = self.linear(x)
+
+        return x
+
+
+
+def resnet18(): return ResNet(BasicBlock, num_block= [2,2,2,2], num_classes= 10)
+
+
```
Comment (varsha-raveendran) [inline src/dlutils/models/ResNets/resnet.py:98]: @abdksyed : we need resnet34 as well right?

**Review by varsha-raveendran — CHANGES_REQUESTED** [review]

File: src/dlutils/models/ResNets/resnet.py (hunk 1 above)
Comment (abdksyed) [reply src/dlutils/models/ResNets/resnet.py:98]: As of now, for current assignment ResNet18 is required, but will add ResNet34 too and push the change.

Comment (abdksyed) [reply src/dlutils/models/ResNets/resnet.py:98]: Added Resnet34!

**Review by kaustubhharapanahalli — APPROVED** [review]

**Comment by varsha-raveendran** [comment]
@kaustubhharapanahalli : yes, we can

**Pull request merged**